server/src/handlers/db_handler.py

```diff
@@ -0,0 +1,32 @@
+from dotenv.main import load_dotenv
+import os
+from pymongo import MongoClient
+
+# Getting the connection string
+load_dotenv()
+connection_string = os.environ["CONNECTION_STRING"]
+
+# Defining a MongoDB client instance
+client = MongoClient(connection_string)
+
+# DB name
+db = client["database1"]
+
+# Referencing the collection
+calendars = db["calendars"]
```

Comment:
This might be a bit of a pain to refactor, but I think its better if we change all references of `calendar` to `course` and `calendars` to `courses` since that is how it's always referred to in the frontend 
https://github.com/techstartucalgary/lifeline/blob/a2af0c59bcdb4dc179b2fae45b187a801a25392b/client/src/logic/icsGen.ts#L3-L21

Reply:
At this point, it's a doable change as we don't have the actual cached courses.